### PR TITLE
Fixes Issue-2767: Windows installation steps to use expo-cli with android studios

### LINKED ISSUE #2767: Launching expo-cli is difficult without instructions
**Where is the problem?**

- [ ] Link **comment/file/somewhere else**
- [ ] Screenshot

**What exactly is wrong:**
I tried to run the expo-cli directory and unfortunately its pretty difficult. I already had to install expo-cli globally, and then also installed android studio. I can get the cli to start running, however, I can't get it to open in a virtual device to see it. The CLI instructions are also not clear enough - I tried to follow these to use android on it.

**What should it be instead:**
I think we need some repro steps - for how others were able to do this on both iOS and Android. This would allow others to become more involved in the project too.

### PR DESCRIPTION
**Pull request by AmasiaNalbandian**
## Issue This PR Addresses
fixes #2767 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [x] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Description
I was not able to launch expo-cli so easily on my PC so I wrote better instructions for Windows. The link provided says the environment variables are only required for MacOS and Linux users but that is not the case, so I outlined all the steps to do that part. 

## Steps to test the PR
- This ones simple so please just review my steps and see if they make sense. Let me know if you have a different experience on windows. 
- If you go to files changed, then click the 3 dots for the file, select view file and then scroll down you can see how it will look in the read.me format on the docs. This will help review.

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Review by cindyledev — DISMISSED** [review]
I love that you added pictures. They help a lot

File: src/docs/docs/tools-and-technologies/react-native.md [hunk 1]
```diff
@@ -38,6 +35,33 @@ For `macOS` more [configuration](https://docs.expo.dev/workflow/android-studio-e
...
+1. [Download Android Studio](https://developer.android.com/studio) using the standard options.
+2. Once you open up Android Studio, navigate to File > Settings
+3. Under Appearance & Behaviour, expand System Settings and select Android SDK. From here select SDK Tools. Install one of the SDK Packages
+   ![SDK Tools](https://i.imgur.com/OXOXFj0.png)
+4. Before hitting apply or OK, note down the Android SDK Location.
+5. Right click your Windows Icon, and select System. Under About you can see the `advanced system settings`, select Advanced and then Environment Variables(as seen below).
+
+   ![Advanced System Settings](https://i.imgur.com/Gdjh1Ix.png)
+
+6. Create a new system variable named `ANDROID_SDK` and the set the value to the path you copied in step 4.
```
Comment (menghif) [inline src/docs/docs/tools-and-technologies/react-native.md:57]: "and then set"
missing an n

**Review by menghif — CHANGES_REQUESTED** [review]
Looks great! There's missing punctuation at the end of some paragraphs and I would also add a blank line between the paragraphs and the images.

**Review by cindyledev — APPROVED** [review]

**Review by menghif — APPROVED** [review]

**Comment by tpmai22** [comment]
@AmasiaNalbandian I regconized that I leave a review but forgot to submit it ! Above is my thought on the change maybe we can figure it out in another PR

File: src/docs/docs/tools-and-technologies/react-native.md [hunk 2]
```diff
@@ -38,6 +35,33 @@ For `macOS` more [configuration](https://docs.expo.dev/workflow/android-studio-e
...
+
+Testing on `iOS` requires `macOS` and [Xcode](https://developer.apple.com/xcode/). Learn how to set up [xCode](https://docs.expo.dev/workflow/ios-simulator/) properly for development.
+
+### Android Studio
+
+On the other hand, [Android Studio](https://developer.android.com/studio?gclid=Cj0KCQiAuvOPBhDXARIsAKzLQ8HSP0n1tojsRjsEZhT2cWmUrps8u4oND0eAONViZsf9rqMk2gFpc_waAtJOEALw_wcB&gclsrc=aw.ds) provides virtual `Android` machine for testing purposes. Learn how to set up [Android Studio](https://docs.expo.dev/workflow/android-studio-emulator/).
+
+##### Windows Installation
+
+1. [Download Android Studio](https://developer.android.com/studio) using the standard options.
```
[the remark this answers is not in the document]
Comment (AmasiaNalbandian) [reply src/docs/docs/tools-and-technologies/react-native.md:48]: how many people tried this on windows without problems?

**Pull request merged**